core/arch/aarch64/aarch64.asm

```diff
@@ -104,19 +104,6 @@ DECL_EXTERN(initstack_mutex)
 DECL_EXTERN(icache_op_struct)
```

Comment:
> The failing tests are not related to the patch.

Please cite the issues covering the failed tests (naming the tests) to prove they are known flakes, ensure all failes we've seen are filed, and give a count of PR's that have hit each flake (each cross-ref is auto-added to the issue).  (Too often new contributors assume something is not related to their own patch just because they think they only touched limited files but don't realize ripple effects.)

Reply:
The x64 failure is #6536 (detach_test).
Previous windows failures have been marked as expected to fail (unit_tests, client.loader).